pinecone/sparse/splade/splade_quora.ipynb

```diff
@@ -0,0 +1,2438 @@
+{
```

Comment:
<blockquote>We first need to initialize our connection to Pinecone to create our vector index. For this, we need a free <a href="https://github.com/pinecone-io/examples/raw/d9d51bddf60c0f6588251c5fccd6165b4d300613/pinecone/sparse/splade/%22https://app.pinecone.io/%22" target="_blank">API key</a>. We initialize the connection like so: </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>